resources/rpm/project.spec.erb

```diff
@@ -118,10 +118,10 @@ done
 %pre
 # Save state so we know later if this is an upgrade or an install
 mkdir -p %{_localstatedir}/lib/rpm-state/%{name}
-if [ $1 -eq 1 ] ; then
+if [ "$1" -eq 1 ] ; then
```

Comment:
This seems odd to me to compare a string to the number. Is this the best way to do this?